client/src/components/new-request/search.vue

```diff
@@ -19,43 +19,48 @@
                    :disabled="request_action_cd !== 'CNV'">
           <template v-slot:activator="scope">
             <div v-on="scope.on">
-              <v-select :error-messages="errors.includes('request_action_cd') ? 'Please select a type' : ''"
+              <v-select :error-messages="errors.includes('request_action_cd') ? 'Please select an action' : ''"
                         :hide-details="!errors.includes('request_action_cd')"
                         :items="requestActions"
                         @change="clearErrors()"
+                        label="Select an Action"
                         filled
                         v-model="request_action_cd">
                 <template slot="item" slot-scope="data">
                   <v-tooltip :disabled="!data.item.blurbs" right transition="fade-transition">
                     <template v-slot:activator="scope">
-                      <span v-on="scope.on" class="list-item">{{ data.item.text }}</span>
+                      <span v-on="scope.on" class="list-item">
+                        {{ data.item.text.replace('Relocate into', 'Relocate into BC') }}
```

Comment:
It's better to not have a lot of code in the template, especially since this is JS instead of TS... specifically, I would like to ensure that "data.item.text" is non-null before chaining "replace", something like `data?.item?.text?.replace(etc)`.

Long story short, if it's a lot of code then use a method instead.

*Maybe* this one is OK if we can guarantee that data.item.text is never null... so then we choose either:
- the code will protect itself from an exception by adding checks, or
- let the code fail, and hope we notice errors when we test, and fix the cause

Reply:
(A lot of this template code will fail if `data.item` is null.)

Reply:
okay good point, I'll add a method for this